Dockerfile

```diff
@@ -53,7 +61,7 @@ RUN mkdir /opt/TrimGalore && \
 # Install Bowtie
 RUN wget -q -O bowtie.zip https://sourceforge.net/projects/bowtie-bio/files/bowtie/1.2.0/bowtie-1.2-linux-x86_64.zip/download && \
   unzip bowtie.zip -d /opt/ && \
-  ln -s /opt/bowtie-1.2/bowtie /usr/local/bin/bowtie && \
+  cp /opt/bowtie-1.2/bowtie* /usr/local/bin/ && \
```

Comment:
Minor thing, but linking is better as it saves a little space.. (but you're right that the original linking statement was broken). This should still work I think:

```bash
ln -s /opt/bowtie-1.2/bowtie* /usr/local/bin/ && \
```